api/notification.proto

```diff
@@ -233,6 +233,7 @@ message EmailServerConfig {
 	google.protobuf.StringValue sender_address = 4;
 	google.protobuf.StringValue server_user = 5;
 	google.protobuf.StringValue server_password = 6;
+	google.protobuf.StringValue sender_addr = 7;
```

Comment:
What is the different from sender_address = 4;

Reply:
did not notice the filed already exists,will fix it.